ting-bill-split.go

```diff
@@ -11,22 +11,144 @@ import (
 	"strconv"
 
 	"github.com/BurntSushi/toml"
+	"github.com/shopspring/decimal"
 )
 
 type bill struct {
-	Minutes   float64 `toml:"minutes"`
-	Messages  float64 `toml:"messages"`
-	Megabytes float64 `toml:"megabytes"`
-	Devices   float64 `toml:"devices"`
-	Extras    float64 `toml:"extras"`
-	Fees      float64 `toml:"fees"`
+	Minutes      float64  `toml:"minutes"`
+	Messages     float64  `toml:"messages"`
+	Megabytes    float64  `toml:"megabytes"`
+	Devices      float64  `toml:"devices"`
+	Extras       float64  `toml:"extras"`
+	Fees         float64  `toml:"fees"`
+	DeviceIds    []string `toml:"deviceIds"`
+	ShortStrawID string   `toml:"shortStrawId"`
+	Total        float64  `toml:"total"`
+}
+
+type billSplit struct {
+	MinSubs   map[string]decimal.Decimal
+	MsgSubs   map[string]decimal.Decimal
+	MegSubs   map[string]decimal.Decimal
+	DeltaSubs map[string]decimal.Decimal
```

Comment:
So, these are the subtotals of each type.  So in `MinSubs` we have how much cost from the minutes charges belong to each deviceId.  The `DeltaSubs` are all the other fees we split evenly, regardless of min/msg/meg usage.

Yeah, this could probably be clearer.  I'm open to suggestions, or I could also try something and see if it helps :D